tools/oesign/oesign.c

```diff
@@ -37,6 +37,14 @@ typedef struct _optional_oe_uuid_t
     oe_uuid_t value;
 } optional_oe_uuid_t;
 
+typedef struct _optional_config_id
```

Comment:
Config_id/config_svn should be set by the enclave loader at the enclave load time. The SGX app select or generate the value based on run time decisions and pass them to the enclave loader. Config_id/config_svn are not decided at Enclave development time, not covered by the signed enclave measurement, nor included in the enclave ELF file. I don't think oesign.c should be affected by config_id/config_svn.